### ISSUE TITLE
Command line/bash

### ISSUE THREAD

**Issue by hakonrossebo** [klaftertief/elm-search#8]
This is a great search engine. Have you considered making this into a command line tool? I think it would be much more practical to have this tool available when I'm using elm-package install

Maybe just writing >elm-search search-term, list results with numbers, press correct number and run elm-package install with the selected package.

**Comment by klaftertief** [comment]
Yes, this is something on my list of things I would like to have. At the moment there is something undocumented in the `server-playground` branch, but this gives you only JSON search results over http.
I'm more interested to provide a CLI for editor integrations, but something manual like you described could be a good starting point...

I can't promise any timeframe at the moment as I want to have some other (organisational and internal) parts settle a bit.

**Comment by hakonrossebo** [comment]
A CLI with editor integrations seems like a very useful tool. Always difficult having enough time :)

**Comment by hakonrossebo** [comment]
Not sure if have time to help out, but you could create some issues if you want help with specific features.

**Comment by klaftertief** [comment]
> Not sure if have time to help out, but you could create some issues if you want help with specific features.

Yes, that's the plan.

**Comment by thenikso** [comment]
Just started checking out Elm and I am shocked that this tool is not on the front page of the official website! 

Why "hoogle" like search for Elm like `elm-search` are not mainstream? I mean http://package.elm-lang.org/ is quite useless if you have no idea on which package might have the function you are looking for... I think.

Anyway I hope I'll be able to help on this particular issue in the future when I'll get fluent with Elm. For the moment 👍

**Comment by andys8** [comment]
Have you thought about integration [elm-console](https://github.com/laszlopandy/elm-console) and integrating existing code for a proof of concept?

**Comment by klaftertief** [comment]
`elm-console` does not work with the current version of Elm. I will get back to implementing a CLI, but there is not timeline...
